### ISSUE TITLE
Removes all js files, including my app's

### ISSUE THREAD

**Issue by Ivy4** [fqborges/webpack-fix-style-only-entries#6]
My entry: 
   
entry: {
        "scripts/index": './source/scripts/index.js',
        "styles/light": './source/styles/light.scss',
        "styles/dark": './source/styles/custom/dark.scss'
    },

It successfully removes the useless light.js and dark.js, but it also removes index.js!

**Comment by fqborges** [comment]
This is unexpected! Could you provide a example project able to reproduce this?

**Comment by fqborges** [comment]
Sorry, misclick.

**Comment by arashohadi** [comment]
This happens to me as well if I have an entry with only one .js file. If I add another one, the plugin does not remove the file.

File app.js **is removed**
`app: [
            './js/app.js'
        ],`

File app.js **is not removed**
`app: [
            './js/app.js',
            './js/anotherFile.js',
        ],`

This only happens on a production build/mode, not while in development mode.

**Comment by dvlden** [comment]
I am using this plugin with Vue CLI v3 and it works perfectly fine. 

Here's my `vue.config.js` file. It might help someone...
* note: by default there's `main.js` file pointed as `app entry`, so we don't specify this one within config, as we're extending it.

```
const FixStyleOnlyEntriesPlugin = require("webpack-fix-style-only-entries")

module.exports = {
  chainWebpack: config => {
    config
      .entry('app')
        .add('./src/assets/styles/app.scss')
        .end()
      .entry('vendor')
        .add('./src/assets/styles/vendor.scss')
      .end()
  },

  configureWebpack: {
    plugins: [
      new FixStyleOnlyEntriesPlugin()
    ]
  }
}
```

Thank you for making this plugin @fqborges - starred. ⭐️

**Comment by brettsmason** [comment]
@fqborges Thanks for the great plugin 😄

I too have encountered this issue. Take a look at this branch for an example: https://github.com/brettsmason/luxe/tree/webpackv2

I have noticed when running `npm run dev` the file `app.js` is present in the `public/js` folder, but if running `npm run production` the file is not there.

Not sure if this helps narrow the issue down or not?

**Comment by fqborges** [comment]
@brettsmason thanks so much. I will get your repository and look at this asap.

**Comment by fqborges** [comment]
fixed on 0.0.5
